### ISSUE TITLE
Add support for android 4.1 and up emoji font

### ISSUE THREAD

**Issue by GoogleCodeExporter** [klausw/hackerskeyboard#376]
```
The summary pretty sums up this; it can be a pain to have to toggle back and 
forth between the dafault keyboard and Hacker's, and I'd rather not get a 
different keyboard since this is so awesome.

```

Original issue reported on code.google.com by `rlrobl...@gmail.com` on 23 Aug 2013 at 5:19

**Comment by moshpirit** [comment]
Yes please, I would totally appreciated some emoji support.

**Comment by Swifty615** [comment]
Yes awesome emoji 
'

**Comment by ascuccaro** [comment]
Are there any news on this? Because being able to access emojis whenever I want is the only reason I'm still using AOSP keyboard (which came preinstalled, obviously, but it's 45 Mb!).

**Comment by kierun** [comment]
+1

**Comment by Yowlen** [comment]
This is probably the one feature that I most want. NextApp's keyboard is the only other one I've found that has CTRL & ALT keys, which I've become dependent on so much that asking me to live without them is like asking a Linux user to start exclusively using Windows. lol. Anyway, they do have the emoji button, but they seem to have abandoned their keyboard and they don't have a compact layout like this one, which makes it extremely difficult to use on my tiny, 4.5" budget phone screen.

**Comment by eXpl0it3r** [comment]
:+1:

Missing this feature is one of reason I'm considering switching to something else.

**Comment by peterjw** [comment]
:+1: I support this feature request

**Comment by treyjazz** [comment]
Yes, please add this - even if as a fork that isn't on the Play Store. I, too, hate the tediousness of having to switch to/from the AOSP keyboard when I want to send emoji.

**Comment by szymonwalle** [comment]
You could also add a "raw Unicode" - you can input hex data and it writes it as a character.

**Comment by eXpl0it3r** [comment]
Totally gonna remember all emojis in hex code. It's not like I can barely remember the one I just wanted to send...

**Comment by peterjw** [comment]
@szymonwalle You're taking the issues title rather verbatim. You can also copy-n-paste emojis, enter emojies by switching keyboards (while typing) and so on. For me (and I guess most supporters here) the issue ist not the _font_, I'am rather interested in a build-in emoji _selector_.

**Comment by nnorm** [comment]
Yes please! Emoji support would be awesome! \o/

**Comment by Viveur** [comment]
Simple guide to add emoji keyboard:

https://github.com/edsilfer/emoji-keyboard/wiki/Development
